front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/HeaderImageDropzone.tsx

```diff
@@ -58,6 +58,12 @@ export const homepageBannerLayoutHeights = {
     tablet: 200,
     phone: 200,
   },
+  fixed_ratio_layout: {
+    // Values, etc. wrong - just added to get FE working with new layout option
+    desktop: 450,
+    tablet: 350,
+    phone: 300,
+  },
```

Comment:
I just copy pasted an existing set of values, so I could get the form to render with this new banner_layout option existing.

Seems to be lots of errors in the browser console, so I doubt I have done this correctly/sufficiently.